### PR TITLE
Feature/conference detail ui

### PR DESCRIPTION
**Pull request by chenry**
This is a rudimentary first draft of the conference detail screen.

### CODE REVIEW

File: Conference-Android/app/src/main/res/layout/activity_conference_detail.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
```
Comment (shekibobo) [inline Conference-Android/app/src/main/res/layout/activity_conference_detail.xml:5]: Prefer one attribute per line for readability and ease of editing.

File: Conference-Android/app/src/main/res/layout/conference_detail.xml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        android:textColor="@color/white" />
+
+    <TextView
+        android:id="@+id/textView3"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="#5F6FCA"
+        android:paddingBottom="10dp"
+        android:paddingLeft="10dp"
+        android:text="May 23 - 24 2015"
```
Comment (shekibobo) [inline Conference-Android/app/src/main/res/layout/conference_detail.xml:31]: Similar to other PR. `@color`, `@dimen`, and `tools:text`.

File: Conference-Android/app/src/main/res/layout/conference_detail.xml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                android:text="Questions?" />
+
+            <TextView
+                android:id="@+id/textView7"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="15dp"
+                android:layout_marginTop="10dp"
+                android:text="Contact: Mahmoud Ahmadinejad" />
+        </LinearLayout>
```
Comment (shekibobo) [inline Conference-Android/app/src/main/res/layout/conference_detail.xml:137]: Hoping for more semantic `id`s for all these views. If not ready for it, I'd recommend adding a `TODO` to remind others to give them better names as the view gets fleshed out more.

**Comment by chenry** [comment]
@shekibobo as always, your input is valued and appreciated.

File: Conference-Android/app/src/main/res/layout/activity_conference_detail.xml (hunk 1 above)
Comment (chenry) [inline Conference-Android/app/src/main/res/layout/activity_conference_detail.xml:5]: done.

File: Conference-Android/app/src/main/res/layout/conference_detail.xml (hunk 2 above)
Comment (chenry) [inline Conference-Android/app/src/main/res/layout/conference_detail.xml:31]: done

**Pull request merged**